src/components/NavigationSection.svelte

```diff
@@ -69,6 +76,22 @@
     }
   }
 
+  function getCategories (items) {
```

Comment:
You do not strengthen your code for the `grouped: items => items[0] instanceof Array` case ?